app/views/data_sources/_source_details.haml

```diff
@@ -8,7 +8,7 @@
       %td
         - if data_span = @data_source.data_span
           = "#{data_span[:start_date]} - #{data_span[:end_date]}"
-    - if (!@data_source.authoritative? || @data_source.hmis?) && can_view_imports? && @data_source.uploads.any? && @data_source.directly_viewable_by?(current_user, permission: :can_view_imports)
+    - if ((!@data_source.authoritative? || @data_source.hmis?) && can_view_imports? && @data_source.uploads.any? && @data_source.directly_viewable_by?(current_user, permission: :can_view_imports)) && can_upload_hud_zips?
```

Comment:
bug fix: the `UploadsController` checks `require_can_upload_hud_zips` - mirror that logic here. Note this condition should probably be moved into the data source policy in future work